### ISSUE TITLE
ValueError: Unsupported Python object None of type <class 'NoneType'>

### ISSUE THREAD

**Issue by NPN** [trailofbits/graphtage#21]
```
$ cat null.json
null
$ graphtage null.json null.json
Traceback (most recent call last):
  File "/sbin/graphtage", line 8, in <module>
    sys.exit(main())
  File "/usr/lib/python3.8/site-packages/graphtage/__main__.py", line 286, in main
    from_tree = from_format.build_tree_handling_errors(from_path, options)
  File "/usr/lib/python3.8/site-packages/graphtage/json.py", line 254, in build_tree_handling_errors
    return self.build_tree(path=path, options=options)
  File "/usr/lib/python3.8/site-packages/graphtage/json.py", line 250, in build_tree
    return build_tree(json.load(f), options)
  File "/usr/lib/python3.8/site-packages/graphtage/json.py", line 70, in build_tree
    raise ValueError(f"Unsupported Python object {python_obj!r} of type {type(python_obj)}")
ValueError: Unsupported Python object None of type <class 'NoneType'>
```
The same happens with JSON files like `[null]` and `{"a": null}`. Version is `0.2.1`.

**Comment by FGRibreau** [comment]
Thanks for your report, I had the same issue (`{"a": null}` case)

**Comment by ESultanik** [comment]
This is fixed in `0.2.2`.
